i18n/readme.md

```diff
@@ -0,0 +1,6 @@
+# About en-US
+
+The locale `en-US` is used to keep the translations up to date when
+uploading the content to crowdin to avoid manually creating the
+different JSON files and dealing with inconsistencies.
+The contents are regenerated periodically.
```

Comment:
Reading through this doc from the lens of someone without context, I have a few questions:

>  to avoid manually creating the different JSON files and dealing with inconsistencies.

Can we link the Docusaurus docs detailing what JSON files are required? Can we list an example of an inconsistency?

> The locale `en-US` is used to keep the translations up to date when uploading the **content** to crowdin [...] The **contents** are regenerated periodically.

Are these two referring to the same contents? What exactly are the contents? What process regenerates the contents, and how often is periodically?

Reply:
> Can we link the Docusaurus docs detailing what JSON files are required? Can we list an example of an inconsistency?

It depends on what plugins we have installed and they also could change if we update docusaurus so there is no "official" list. Does that make sense?

> Are these two referring to the same contents? What exactly are the contents? What process regenerates the contents, and how often is periodically?

The contents of the folder are these JSON files. I've added a pre-commit hook that checks if `docusuaurus.config.js` or `sidebars.js` have been modified and then forces the regeneration.

I'll make all of this clearer 😊